src/taipy/core/data/data_node.py

```diff
@@ -359,7 +359,7 @@ def filter(self, operators: Union[List, Tuple], join_operator=JoinOperator.AND):
         join operator (_AND_ or _OR_).
 
         Parameters:
-            operators (Union[List[Tuple], Tuple]): TODO
+            operators (Union[List[Tuple], Tuple]): 3-tuples or list of 3-tuples (key, value, `Operator^`).
```

Comment:
```suggestion
            operators (Union[List[Tuple], Tuple]): A 3-element tuple or a list of 3-element tuples, each is in the form of (key, value, `Operator^`).
```
This is unrelated to the purpose of the PR. But I think this is clearer?

I'm not sure if I understand it correctly or not though, but we need to make it clearer for the user.